test/base64/test_base64.rb

```diff
@@ -112,4 +112,13 @@ def test_urlsafe_decode64_unpadded
     assert_equal("\0\0\0", Base64.urlsafe_decode64("AAAA"))
     assert_raise(ArgumentError) { Base64.urlsafe_decode64("AA=") }
   end
+
+  def test_aliases
+    assert_equal(Base64.method(:encode), Base64.method(:encode64))
+    assert_equal(Base64.method(:decode), Base64.method(:decode64))
+    assert_equal(Base64.method(:strict_encode), Base64.method(:strict_encode64))
+    assert_equal(Base64.method(:strict_decode), Base64.method(:strict_decode64))
+    assert_equal(Base64.method(:urlsafe_encode), Base64.method(:urlsafe_encode64))
+    assert_equal(Base64.method(:urlsafe_decode), Base64.method(:urlsafe_decode64))
+  end
 end
```

Comment:
I would like to add a test to make sure the aliases are not the target of `include`.

```suggestion

  class Test
    include Base64
    def check(name)
      respond_to?(name, true)
    end
  end

  def test_module_function_test
    assert_equal(true, Test.new.check(:encode64))
    assert_equal(false, Test.new.check(:encode))
  end
end
```